lbrynet/lbrynet_daemon/Publisher.py

```diff
@@ -48,6 +48,7 @@ def publish_stream(self, name, file_path, bid, metadata):
         metadata['content_type'] = get_content_type(file_path)
         metadata['ver'] = Metadata.current_version
         claim_out = yield self.make_claim(name, bid, metadata)
+        yield self.lbry_file.start()
```

Comment:
I think this calls 
https://github.com/lbryio/lbry/blob/master/lbrynet/cryptstream/client/CryptStreamDownloader.py#L97, which gives you back a finished deferred - and you're going to wait for that to be called. 

This seems odd to want to 'start' downloading a stream that you already have.  If you want to populate variables, it would be better to have a function that does just that and have `start()` also call that function.